PowerShell/ScubaGear/Testing/Unit/PowerShell/Providers/EXOProvider/Select-DohServer.Tests.ps1

```diff
@@ -0,0 +1,34 @@
+$ProviderPath = "../../../../../Modules/Providers"
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath "$($ProviderPath)/ExportEXOProvider.psm1") `
+    -Function 'Select-DohServer' -Force
+
+InModuleScope 'ExportEXOProvider' {

```

Comment:
I would like to see some basic documentation describing the logic paths that these unit tests are meant to exercise. I can reverse engineer it from looking at the code for Select-DohServer and then reading the unit tests, but it would be better to have some documentation. Nothing too crazy. I will make this point at the stand-up so that others can live up to the same expectation. Thoughts?

Reply:
I won't deny that more documentation would be helpful, but I'm not sure exactly what to put. Would something like this be helpful?
![image](https://github.com/cisagov/ScubaGear/assets/106177711/916a0b6b-4527-48da-a5d4-4277cdafc9f8)